roles/auth_proxy/tasks/main.yml

```diff
@@ -1,6 +1,18 @@
 ---
 # tasks file for auth_proxy
 
+# Load the auth-proxy-image from local tar. Ignore any errors to handle the
+# case where the image is not built in
+- name: copy auth-proxy image
+  copy: src={{ auth_proxy_binaries }}/auth-proxy-image.tar dest=/tmp/auth-proxy-image.tar
+  when: auth_proxy_local_install == True
+  ignore_errors: yes
```

Comment:
Let's eliminate the ignore_errors for all such cases as we have the conditional var.